Helper/AutomatedTesting.php

```diff
@@ -165,6 +214,53 @@ public function __construct(
         $this->stockRegistry = $stockRegistry;
     }
 
+    /**
+     * Retrieve an Order from the repository
+     *
+     * @return Order|null
+     */
+    protected function getPastOrder()
+    {
+        try {
```

Comment:
can we not do try catch in this? cuz there's already a try catch in the main function

Reply:
done